requirements.txt

```diff
@@ -1,3 +1,10 @@
+
+            Warning: The lock flag -r/--requirements will be deprecated in a future version
```

Comment:
Curious how if this warning was added manually or by some kind of tooling. Does it not need to be inside a `#`-style comment?

Reply:
This file is generated here:

https://github.com/coda/packs-sdk/blob/c3a47e4bfc19ac0fe9bb2da11f02fd31fcb6d7c6/Makefile#L42

Readthedocs requires it (it doesn't support Pipfile). Looks like this does [break the Readthedocs build](https://readthedocs.com/projects/coda-packs-sdk/builds/934991/).

Reply:
Oops, sorry about that, removed for now. I'll take a stab in a different PR.

Reply:
https://github.com/coda/packs-sdk/pull/1972